### PR TITLE
fix(snippet): fix incorrect indentation closes #96

### LINKED ISSUE #96: Wrong flutter command path after updating
**Describe the bug**
I've installed `coc-flutter` on December and didn't update it until yesterday. I didn't modify any option regarding to executables or anything. As the title says, the flutter command path detected not correct, as shown in the logs later on
**To Reproduce**
My coc-settings.json flutter related lines:
```json
  "flutter.lsp.initialization.closingLabels": true,
  "flutter.lsp.debug": true,
  "flutter.trace.server": "verbose",
```
flutter --version
```
$ flutter --version
Flutter 1.22.6 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 9b2d32b605 (5 weeks ago) • 2021-01-22 14:36:39 -0800
Engine • revision 2f0af37152
Tools • Dart 2.10.5
```

flutter doctor
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.6, on Linux, locale en_US.UTF-8)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[!] Android Studio (version 4.0)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.
```
Execute any flutter command (flutter pub get on save, :CocCommands flutter.devices..)

**Expected behavior**
I expect it to work as usual it did before, not specifying the path manually 
**Screenshots**
image
image

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Vim or Neovim: Neovim
 - (Neo)vim version: v0.4.4

**Output channel:**
```
[dev-command]: register dev command
[sdk]: which flutter command => /var/lib/snapd/snap/bin/flutter
[sdk]: flutter command path => /usr/bin/snap/bin/flutter
[sdk]: dart sdk home => /usr/bin/cache/dart-sdk
[sdk]: which dart command => /usr/bin/dart
[sdk]: dart command path => /opt/dart-sdk/bin/dart
[sdk]: dart sdk home => /opt/dart-sdk
[sdk]: analyzer path => /opt/dart-sdk/bin/snapshots/analysis_server.dart.snapshot
[sdk]: dart command path => /opt/dart-sdk/bin/dart
[fs]: executing command /opt/dart-sdk/bin/dart --version
[sdk]: dart version: v2.10.4
[lsp-server]: rightVersion true
[global-commands]: pub get command, workspace: /home/username/project-path
[fs]: executing command /usr/bin/snap/bin/flutter pub get
[fs]: executing command /usr/bin/snap/bin/flutter emulators
[dev-command]: run dev server, devServer state: false
[server]: server start at: /home/username/project-path
```

### PR DESCRIPTION
**Pull request by iamcco**
relates to #95

### CODE REVIEW

File: src/server/lsp/resolveCompleteItem.ts [hunk 1]
```diff
@@ -36,7 +36,8 @@ export const getResolveCompleteItemFunc = (options: { completeFunctionCalls: boo
   }
 
   // improve setState
-  if (label === 'setState(() {});' && insertTextFormat !== InsertTextFormat.Snippet) {
+  // refer: https://github.com/iamcco/coc-flutter/issues/95
+  if (label === 'setState(() {});') {
     item.insertText = ['setState(() {', '\t${1}', '});${0}'].join('\n');
```
Comment (Kavantix) [inline src/server/lsp/resolveCompleteItem.ts:41]: This one also has indentation, shouldn't we remove it here then as well.

**Comment by Kavantix** [comment]
The issue was only about `setState` but this happens with basically most (if not all) of the snippets

File: src/server/lsp/resolveCompleteItem.ts (hunk 1 above)
Comment (iamcco) [reply src/server/lsp/resolveCompleteItem.ts:41]: `['setState(() {', '\t${1}', '});${0}'].join('\n')` will extend to 

```
setState(() {
  |
});
```

the indentation is necessary

**Comment by iamcco** [comment]
> The issue was only about `setState` but this happens with basically most (if not all) of the snippets

If we want to resolve all snippets, we should remove all the indentation of all snippets.

File: src/server/lsp/resolveCompleteItem.ts (hunk 1 above)
Comment (Kavantix) [reply src/server/lsp/resolveCompleteItem.ts:41]: 👍🏻

**Pull request closed without merging**